contracts/distribution/Distribution.sol

```diff
@@ -266,6 +276,14 @@ abstract contract Distribution is
             data);
     }
 
+    function _withdrawFees(Amount amount)
```

Comment:
why not in contract Component? if no real reason move to Component and refactor Pool, Product accordingly